cmd/zcertificate/main.go

```diff
@@ -42,19 +43,22 @@ func init() {
 	flag.IntVar(&workers, "workers", 1, "Specifies number of goroutines to use to parse and lint certificates.")
 	flag.IntVar(&numProcs, "procs", 0, "Specifies number of processes to run on. Default is 0, meaning use current value of $GOMAXPROCS.")
 	flag.BoolVar(&crashIfParseFail, "fatal-parse-errors", false, "Halt if a certificate cannot be parsed. Default is to log.")
+	flag.BoolVar(&jsonIfParseFail, "json-parse-errors", false, "Output json if a certificate cannot be parsed. Default is not to.")
```

Comment:
It seems like if `-fatal-parse-errors=true` is specified alongside `-json-parse-errors=true` then the former will take precedence and no JSON output will be provided. Can you check for this case of incompatible flags in `main` and return a meaningful error to the user?

Reply:
Can you also update `README.md` to add this new flag in the `-h` output there?

Reply:
Done